src/code/stores/graph-store.coffee

```diff
@@ -529,32 +529,43 @@ GraphStore  = Reflux.createStore
       model: modelDescription
     }
 
-  # Returns the minimum complexity that the current graph allows.
+  # Returns the minimum simulation type that the current graph allows.
   # Returns
   #   0 (diagramOnly)    if there are no defined relationships
-  #   1 (basic)          if all scalars are `about the same`
-  #   2 (expanded)       if there are no collectors
-  #   3 (collectors)     if there are collectors
-  getMinimumComplexity: ->
-    minComplexity = 0
+  #   1 (static)         if there are no collectors
+  #   2 (time)           if there are collectors
+  getMinimumSimulationType: ->
+    minSimulationType = 0
 
     links = @getLinks()
-    _.each links, (link) ->
-      return unless (source = link.sourceNode) and (target = link.targetNode)
+    for link in links
+      continue unless (source = link.sourceNode) and (target = link.targetNode)
 
       if source.isAccumulator or target.isAccumulator
-        # we know we have to be the highest complexity
-        minComplexity = 3
+        # we know we have to be time-based
+        return 2
```

Comment:
Minor nitpick: I would rather read `return AppSettingsStore.store.SimulationType.time` used throughout.

Reply:
(or a shorter local name)

Reply:
:+1: You're right, fixed.